Rx.playground/Pages/Combining_Operators.xcplaygroundpage/Contents.swift

```diff
@@ -58,7 +58,8 @@ example("merge") {
 /*:
  ----
  ## `zip`
- Combines up to 8 source `Observable` sequences into a single new `Observable` sequence, and will emit from the combined `Observable` sequence the elements from each of the source `Observable` sequences at the corresponding index. [More info](http://reactivex.io/documentation/operators/zip.html)
+ Combines up to 8 source `Observable` sequences into a single new `Observable` sequence and will emit one element per full set of elements emitted by it's source sequences and not before receiving at least one element per each of its source sequences.
```

Comment:
Hi @Nirma ,

Maybe we could improve this initial description, but I'm not sure this is an improvement.

The suggested emphasizes first elements, which shouldn't be the case.

The meaning of "full set of elements" is also vague.